### ISSUE TITLE
The Evil Within (268050)

### ISSUE THREAD

**Issue by arzardk** [ValveSoftware/Proton#236]
I launched game, Bethesda logo and others plays, after that loading screen appears and game hangs after 2 seconds without any crash message.

[System specs](https://gist.github.com/arzardk/6f020699814adc78c0b9c8c1e1ee2764)
[steam-268050.log](https://github.com/ValveSoftware/Proton/files/2313447/steam-268050.log)
![2018-08-23 10-44-33](https://user-images.githubusercontent.com/37209942/44512257-9779ac80-a6c2-11e8-865f-7001f2b1806d.png)

I managed to run this game in windowed mode which can be toggled in configure file located in this directory: `/home/arzardk/.steam/steam/steamapps/compatdata/268050/pfx/drive_c/users/steamuser/Saved Games/TangoGameworks/The Evil Within/base`

**Comment by tr37ion** [comment]
You can switch fullscreen/window mode via `ALT+ENTER` at main menu or check upper config file for settigns like:
```
...
win_ypos "5"
win_xpos "1912"
in_joystick "1"
s_volume_music "-7.958801"
r_swapInterval "-1"
r_windowPosY "28"
r_windowPosX "1916"
r_windowHeight "1008"
r_windowWidth "1920"
r_mode "23"
r_screenGrainAmount "1"
r_toneGamma "2.1"
r_lpp_SMAA "1"
r_lpp_FXAA "0"
r_lpp_CameraBlur "0"
r_letterBoxMode "0"
con_IconWindowY "24"
con_IconWindowX "24"
sys_lang "english"
r_forceAspectRatio "1.7777"
```

**Comment by zzhiyi** [comment]
Proton 3.16 should have some improvement. The main menu doesn't hang for me but it's indeed sluggish. However, that could be related to that I'm using the integrated graphics card, producing only several fps.

**Comment by flibitijibibo** [comment]
Seems this got reintroduced on or after 4.11-4. Disabling fsync/esync fixes the issue for me.

**Comment by AwesamLinux** [comment]
**System Information**
- GPU: RX 5700
- GPU Driver: Mesa 20.0.0-devel (git-472593e 2020-01-08 bionic-oibaf-ppa)
- Kernel version: 5.3.18-050318-lowlatency
- Distro version: 19.3 Tricia (64 bit) (Cinnamon)
- Proton version: 4.11-11

**Compatibility Report**

Everything worked out of the box with good performance:
✔ Keyboard & Mouse controls 
✔ Switching between fullscreen and desktop 
✔ Fullscreen mode
✔ Loading game saves 
✔ Nintendo Switch Pro Controller 
✔ Good performance
✔ No rendering issues with ACO
✔ Videos

So for me, this is looking like it could be whitelisted

The log created by the game is huge so perhaps there is some issue that did not affect the user experience.
[steam-268050.log.zip](https://github.com/ValveSoftware/Proton/files/4041652/steam-268050.log.zip)
[system_info.txt](https://github.com/ValveSoftware/Proton/files/4041653/system_info.txt)

**Comment by kentosama** [comment]
I have the same problem after upgrading to Debian testing. I upgraded to Debian unstable and the game still hangs on startup.

![Capture d’écran du 2020-03-20 13-23-29](https://user-images.githubusercontent.com/3535658/77163279-080aca00-6aae-11ea-9d0b-47e5879da360.png)

### System information
- GPU: R9 280X
- GPU Driver: Mesa 20.0.2
- Kernel version: 5.4.0-4-amd64
- Distro version: Debian GNU/Linux bullseye/sid (64 bits) (Cinnamon)
- Proton version: 5.0-4

[system_info.txt](https://gist.github.com/kentosama/7bd858464de65f4d666b032d2aaa4a45)
[steam_268050.log](https://gist.github.com/kentosama/6b3f41f5b9fd84ad4b39b4129bd60d35)

**Comment by kisak-valve** [comment]
Hello @kentosama, can you check if setting the game's launch options to `PROTON_NO_ESYNC=1 %command%` has an effect?

**Comment by kentosama** [comment]
Hello @kisak-valve with `PROTON_NO_ESYNC=1 ` the game still hangs on loading screen.

**Comment by kentosama** [comment]
After install ArchLinux, the game run without problem with Mesa 20.

**Comment by luca-s** [comment]
[I am adding this report just for future reference]

I have a similar issue, the hame hangs forever at start-up, I can only see a black screen. The issue is related only to proton 5.13-4 and the game works fine with proton 5.0-10 instead (months ago I also played the game 90+ hours with proton 5.0-7 without issues).

EDIT: added system info and logs links

[System info](https://gist.githubusercontent.com/luca-s/ea2c8855d59411524c84a3aebfa7f540/raw/0ba3464e8cf309a0ae96e58ce951cd8fe748596f/system-info.txt)
[proton 5.13-4 log - black screen](https://gist.githubusercontent.com/luca-s/ea2c8855d59411524c84a3aebfa7f540/raw/0ba3464e8cf309a0ae96e58ce951cd8fe748596f/steam-268050-proton5.13-4%2520(black%2520screen).log)
[proton 5.0-10 log - works fine](https://gist.githubusercontent.com/luca-s/ea2c8855d59411524c84a3aebfa7f540/raw/0ba3464e8cf309a0ae96e58ce951cd8fe748596f/steam-268050-proton5.0-10%2520(works%2520fine).log)

**Comment by kisak-valve** [comment]
Hello @luca-s, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by luca-s** [comment]
@kisak-valve updated my previous post with links to the files (thanks)

**Comment by kisak-valve** [comment]
Thanks @luca-s, I think your system information is hinting that you're seeing https://github.com/ValveSoftware/steam-runtime/issues/312. Can you try temporarily disabling mesa/ANV with something like `sudo mv /usr/share/vulkan/icd.d/intel_icd.x86_64.json /usr/share/vulkan/icd.d/intel_icd.x86_64.json.disabled` and test if that has an effect?

**Comment by luca-s** [comment]
You are right @kisak-valve , this is the same issue of  [ValveSoftware/steam-runtime#312](https://github.com/ValveSoftware/steam-runtime/issues/312). If I rename the file I can play the game.
